packages/itwin/tree-widget/src/tree-widget-react/components/trees/categories-tree/CategoriesTreeComponent.tsx

```diff
@@ -42,6 +43,7 @@ interface CategoriesTreeComponentProps
    * ```
    */
   headerButtons?: Array<(props: CategoriesTreeHeaderButtonProps) => React.ReactNode>;
+  treeWidgetViewport?: TreeWidgetViewport;
```

Comment:
I'd say this should be changed to `viewport?: Viewport | TreeWidgetViewport`. Also, let's document what it means and what happens when it's not provided.

Reply:
same for other tree components